engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1858,6 +1854,10 @@ static void LogFrameBufferError(GLenum status)
 
         OpenGLContext* context = (OpenGLContext*) _context;
 
+    #if !defined(GL_ES_VERSION_2_0)
+        glBindVertexArray(context->m_GlobalVAO);
+    #endif
```

Comment:
We must refresh the binding table at "some point" during our own rendering pipeline, and binding vertex declarations happens more rarely than binding vertex buffers.